docs/docs/auth/dbauth.md

```diff
@@ -352,11 +357,15 @@ To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js`
 
 ```json title="api/src/functions/auth.js"
 cookie: {
-  HttpOnly: true,
-  Path: '/',
-  SameSite: 'Strict',
-  Secure: process.env.NODE_ENV !== 'development' ? true : false,
-  Domain: 'example.com'
+  attributes: {
+    HttpOnly: true,
+    Path: '/',
+    SameSite: 'Strict',
+    Secure: process.env.NODE_ENV !== 'development' ? true : false,
+    // highlight-next-line
+    Domain: 'example.com'
```

Comment:
The text above says "set the `cookie.Domain` property". Should that also be updated to say `cookie.attributes.Domain`? Or is it better to just say "set the `Domain` property", and not care exactly how it's nested?

Reply:
Good catch :+1:  → 978ad2b6c3b59363fb8fd34852e369ecceb478bb